gulp/.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I'm thinking that we should have ESLint at the root of the rig for all of the JS scripts including gulp.  Thoughts?

Reply:
@hellofromtonya the issue I see with that is the `env:node`, which is true for gulp but not the JS being written for the front end scripts. Without that ESLint throws warnings for using `process` and other Node variables in gulp.

We could do inline ESLint ignore comments in the gulp files to get around that or maybe make creative use of the [ESLint configuration](https://eslint.org/docs/user-guide/configuring) file precedence.

Reply:
Yup, you're right @ataylorme.  Let's do this to keep it clean.  Leave it as you have it.  Then we can take on a new issue for how we want to proceed with adding ESLint to the root of the rig.  Sound good?

Reply:
Works for me. There was no ESLint config in WP Rig but I wanted to have one to help me write better code when working with gulp for this PR

Reply:
The ESLint config sits in `package.json` https://github.com/wprig/wprig/blob/8bfd9cbe4f7055df2ea7d519666c77803bab1528/package.json#L62